Jenkinsfile

```diff
@@ -1,22 +1,33 @@
-pipeline {
-    agent any
-    stages {
-        stage("Test and Package") {
+pipeline { 
+    agent any 
+    
+    tools { 
+	    maven 'Maven 3.8.1' 
+	    jdk 'jdk16' 
```

Comment:
Do we need jdk16 for this? I don't think we have this configuration in our jenkins at the moment. (unless you updated the ami since I last used it). If so, let me know and we can make some changes to the most up-to-date one